posthog/models/async_deletion/delete_events.py

```diff
@@ -49,10 +51,13 @@ def process(self, deletions: List[AsyncDeletion]):
             ON CLUSTER '{CLICKHOUSE_CLUSTER}'
             DELETE
             WHERE
-                joinGet({temp_table_name}, 'id', team_id, 0, toString(team_id)) > 0 OR
-                joinGet({temp_table_name}, 'id', team_id, 1, toString(person_id)) > 0
+                team_id IN %(team_ids)s AND
```

Comment:
I like this because it gives us a second level of safety when deleting from this table